AndelaOSP/art-android
# #156266679 Security user logout

## What does this PR do?
It adds the logout functionality for a security user.
## Description of Task to be completed?
- Add logout UI tests
- Create the logout dialog and logic 
## How should this be manually tested?
- Run the SettingsTest test file
- Launch the App on a device and login to access the security dashboard. Select the settings menu item so that you are directed to the settings activity. Clicking the logout text view will present the logout dialog from which you can confirm or cancel logout.
## What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/156266679
## Screenshot
Logout dialog
<img width="353" alt="screen shot 2018-04-04 at 13 38 19" src="https://user-images.githubusercontent.com/29823251/38303290-4af5d7f8-380e-11e8-869c-cece5f94f139.png">
